Packages/Sources/DataModel/APIParameters.swift

```diff
@@ -0,0 +1,11 @@
+import Tagged
+
+public enum Parameters {
+    public enum Key {}
+    public enum ETH {}
+    public enum Page {}
+}
+
```

Comment:
I have been thinking about this problem in a persona project where I have CloudKit records, Core Data entities, and Entity summary structs. Using recordName string as an object identifier when fetching entities from Core Data. Used the opportunity to experiment from a clean slate 🤷‍♂️